docs/builds.md

```diff
@@ -86,6 +86,34 @@ to use the network as source of truth for authentication, using e.g. [FreeIPA](h
 That avoids the need to hardcode any users or keys in the image, just the
 setup necessary to contact the IPA server.
 
+### Avoiding home directory persistence
+
+In a default installation, the `/root` and `/home` directories are persistent,
+and are symbolic links to `/var/roothome` and `/var/home` respectively.
+This persistence is typically highly desirable for machines that are somewhat "pet" like,
+from desktops to some types of servers, and not at all desirable for others.
+
+It's recommended for most use cases that don't want a persistent home
+directory to inject a systemd unit like this for both these directories, for
+example:
+
+```systemd
+[Unit]
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=swap.target
+
+[Mount]
+What=tmpfs
+Where=/var/home
+Type=tmpfs
+```
```

Comment:
I'd like to see an explanation of how this solves the described persistence issue.  I can interpret the tmpfs trick but assume that users who don't now the solution already may have troubles interpreting the unit.

Reply:
I added a `Description` to the unit.

Reply:
Can we make this more obvious?  I feel like this is the important part of this doc section.  What I am looking for explicitly is the "stating the obvious" rule from https://www.amazon.com/BUGS-Writing-Revised-Guide-Debugging/dp/020137921X.  Probably just adding that a tmpfs won't survive a reboot, so changes are non-persistent.

Reply:
Hmm.  I have a baseline assumption in general that the target audience for bootable containers is medium-to-advanced Linux users.  For example, we aren't going to explain things like partitions, SSH keys, packages, or systemd units.  That doesn't mean every user knows *all* of those things, but if they don't we expect them to figure out how to find out more.

`tmpfs` feels to me very much like one of those things...and if you don't know what tmpfs is, pasting it into Google right now leads to https://www.kernel.org/doc/html/latest/filesystems/tmpfs.html which explains it clearly right?

The other perhaps notable thing here is these docs are likely to get rewritten anyways for integration into downstreams.

Reply:
That said, it's easy to hyperlink that, so done.